docs/manager.md

```diff
@@ -86,37 +86,50 @@ NB: we set environment variables that we will use in the shell process where we
 
 ## Deployment
 
-To start the service, execute the following shell script (note a server needs to be running see [here](https://github.com/ultravioletrs/cocos/tree/main/test/computations)):
+To start the service, execute the following shell script (note a server needs to be running see [here](https://github.com/ultravioletrs/cocos/tree/main/test/computations)). Additionally, the required environment variables should be set in [cocos-manager.env](https://github.com/ultravioletrs/cocos/blob/main/cocos-manager.env) beforehand.
 
 ```bash
 # download the latest version of the service
 go get github.com/ultravioletrs/cocos
 
 cd $GOPATH/src/github.com/ultravioletrs/cocos
+```
 
-# compile the manager
-make manager
+Variables should be set at this point in [cocos-manager.env](https://github.com/ultravioletrs/cocos/blob/main/cocos-manager.env).
 
-# copy binary to bin
+```bash
+# builds binaries, copies them to bin and installs the env variables required for manager
 make install
 
-# set the environment variables and run the service
-MANAGER_GRPC_URL=localhost:7001
-MANAGER_LOG_LEVEL=debug \
-MANAGER_QEMU_USE_SUDO=false \
-MANAGER_QEMU_ENABLE_SEV=false \
-./build/cocos-manager
+# run the service
+make run
 ```
 
-To enable [AMD SEV](https://www.amd.com/en/developer/sev.html) support, start manager like this
+To run manager service without using makefile:
 
-```sh
-MANAGER_GRPC_URL=localhost:7001
-MANAGER_LOG_LEVEL=debug \
-MANAGER_QEMU_USE_SUDO=true \
-MANAGER_QEMU_ENABLE_SEV=true \
-MANAGER_QEMU_SEV_CBITPOS=51 \
-./build/cocos-manager
+```bash
+sudo systemctl start cocos-manager
+```
+
+To enable [AMD SEV](https://www.amd.com/en/developer/sev.html) support, start manager after settin these environment variables in [cocos-manager.env](https://github.com/ultravioletrs/cocos/blob/main/cocos-manager.env).
+
+```bash
+MANAGER_QEMU_USE_SUDO=true
+MANAGER_QEMU_ENABLE_SEV=true
+MANAGER_QEMU_SEV_CBITPOS=51
+```
+
+Then install the service:
+
+```bash
+# install the service with the environment variables
+make install
+
+# start the service
+make run
+
+# you can also start the service with
+sudo systemctl start cocos-manager
```

Comment:
just focus on the make commands, if something is missing we can add a make command, for example one to restart manager